docs/README.md

```diff
@@ -130,3 +130,44 @@ Most of the site's images are screenshots of Dagit. There is a semi-automated sy
 ## Navigation schema
 
 If you are adding a new prose page or want to update the navigation in the sidebar, update the `docs/content/_navigation.json` file. The structure is self-explanatory when looking at the sidebar.
+
+## Troubleshooting
+
+### Problem: ModuleNotFoundError: No module named 'X'
+
+Example stack trace:
+
+```
+dagster/docs $ make apidoc-build
+...
+
+/Users/jamie/dev/dagster/docs/sphinx/sections/api/apidocs/cli.rst:60: ERROR: Failed to import "grpc_command" from "dagster._cli.api". The following exception was raised:
+Traceback (most recent call last):
+File "/Users/jamie/dev/dagster/docs/.tox/sphinx/lib/python3.9/site-packages/sphinx_click/ext.py", line 403, in _load_module
+    mod = __import__(module_name, globals(), locals(), [attr_name])
+File "/Users/jamie/dev/dagster/python_modules/dagster/dagster/_cli/__init__.py", line 8, in <module>
+    from .api import api_cli
+File "/Users/jamie/dev/dagster/python_modules/dagster/dagster/_cli/api.py", line 14, in <module>
+    from dagster._cli.workspace.cli_target import (
+File "/Users/jamie/dev/dagster/python_modules/dagster/dagster/_cli/workspace/__init__.py", line 1, in <module>
+    from .cli_target import get_workspace_process_context_from_kwargs, workspace_target_argument
+File "/Users/jamie/dev/dagster/python_modules/dagster/dagster/_cli/workspace/cli_target.py", line 7, in <module>
+    import tomli
+ModuleNotFoundError: No module named 'tomli'
+```
+
+Solution:
+
+You likely need to rebuild your tox environment. To do this, run:
+
+```
+dagster/docs $ tox -r
+```
+
+to rebuild the environment for every tox command, or run:
+
+```
+dagster/docs $ tox -re sphinx
+```
+
+to rebuild and run just the sphinx command
```

Comment:
nit
```suggestion
**Solution:**

You likely need to rebuild your tox environment. In the `dagster/docs` directory, run one of the following:

```bash
# Rebuild the environment for every tox command
tox -r

# Rebuild the environment for a specific command (e.g. sphinx)
tox -re sphinx
```